testutils/assert/assert.go

```diff
@@ -0,0 +1,57 @@
+package assert
```

Comment:
we could, although the whole concept of vendoring dependencies makes me ill, I have to deal with it in our everyday Go development and I can't wait until we switch to a sane dep mgr. I've learned to appreciate Python's virtualenv.